requirements.txt

```diff
@@ -1,7 +1,6 @@
 # Django/Framework Packages
-django==1.4.12
+django>=1.8,<1.9
```

Comment:
Oops, my mistake, this is a library, not an IDA, so we should not pin a version.